packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -167,9 +162,6 @@ export class VaTextInput {
   };
 
   render() {
-    const describedBy =
-      `${this.ariaDescribedby} ${this.error ? 'error-message' : ''}`.trim() ||
-      null; // Null so we don't add the attribute if we have an empty string
```

Comment:
Do we want to remove the `aria-describedby` even when there is an error message? This seems like it could be left while removing the external-facing prop.

Reply:
@bkjohnson I have added that portion back in